Dockerfile

```diff
@@ -14,6 +14,11 @@ RUN bin/bash -l -c "gem install aws-sdk mime-types"
 RUN apt-get install -y python2.7
 ENV PYTHON /usr/bin/python2.7
 
+# install hugo
+ENV HUGO_VERSION 0.21
```

Comment:
We have a working branch of our site in `HUGO` https://github.com/GSA/digital.gov/tree/replatform-hugo
Let us know if it would be better to store this in a separate repo for testing in Federalist